### PR TITLE
Create key codes object and replace body-parser

### PR DESCRIPTION
**Pull request by ElCholoGamer**
- Made a `codes` object for storing keys and their control state names.
- Replaced `body-parser` with `express.json` to reduce dependencies.

### CODE REVIEW

**Comment by AwesomestCode** [comment]
TBH I don't think a keys object is necessary.

**Comment by rom1504** [comment]
Can you run npm run fix to fix the lint ?

**Comment by ElCholoGamer** [comment]
Alright, fixed.

File: package.json [hunk 1]
```diff
@@ -1,76 +1,75 @@
...
-        "process": "^0.11.10",
-        "standard": "^16.0.3",
-        "stream-browserify": "^3.0.0",
-        "three": "^0.124.0",
-        "timers-browserify": "^2.0.12",
-        "webpack": "^5.11.0",
-        "webpack-cli": "^4.2.0",
-        "webpack-dev-server": "^3.11.0"
-    }
+	"name": "prismarine-web-client",
```
Comment (rom1504) [inline package.json:2]: Looks like your ide reformated this to have 4 spaces indentation instead of 2.
Can you go back to 2 space indentation ?

Comment (ElCholoGamer) [reply package.json]: Yes, fixed.

**Review by rom1504 — APPROVED** [review]

**Pull request merged**